adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/net45/UserPasswordCredential.cs

```diff
@@ -85,13 +85,13 @@ internal override void ApplyTo(DictionaryRequestParameters requestParameters)
             requestParameters[OAuthParameter.GrantType] = OAuthGrantType.Password;
             requestParameters[OAuthParameter.Username] = this.UserName;
             requestParameters[OAuthParameter.Password] = new string(PasswordToCharArray());
-            
-            if (SecurePassword != null && !SecurePassword.IsReadOnly())
-            {
-                SecurePassword.Clear();
-            }
 
-            SecurePassword = null;
+            if (SecurePassword != null && !SecurePassword.IsReadOnly()) 
```

Comment:
Did you intend to have this change in this PR?

Reply:
+1

Reply:
That is not in the PR...

Reply:
I removed it... by re-introducing it in a later commit. not sure what you mean.